.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on: [push, pull_request]
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: "1.17.8"
+    - run: |
+        git config --global user.email "github-action@etcd.io"
+        git config --global user.name "Github Action"
+        gpg --batch --gen-key <<EOF
+        %no-protection
+        Key-Type: 1
+        Key-Length: 2048
+        Subkey-Type: 1
+        Subkey-Length: 2048
+        Name-Real: Github Action
+        Name-Email: github-action@etcd.io
+        Expire-Date: 0
+        EOF
+        BRANCH=main ./scripts/release.sh --no-upload --no-docker-push 3.6.99
```

Comment:
Just curious about the value, 3.6.99? Thanks!

Reply:
I assume it's just value bigger than any 3.6 patch version released so far.

Reply:
Needed to pick a version to release script to run. I used `3.6` as I plan to add tests for other branches in next PRs. Picked `.99` as wanted the version to correct but easy to distinguish from normal release. We could pick any other version that would fulfills those requirements.

Reply:
Sounds good. Thanks @serathius